tests/src/FunctionalJavascript/Form/MigrationFormTest.php

```diff
@@ -112,8 +153,7 @@ public function testAddMigrationForTaxonomyTerm() {
    * Tests editing an existing migration.
    */
   public function testEditMigration() {
-    // Create two vocabularies.
-    $vocabulary1 = $this->createVocabulary();
+    // Create vocabulary.
```

Comment:
I see that in ::testEditMigration() you removed the line of creating the first vocabulary. That line was there on purpose. By default, the first vocabulary is preselected if you don't specify a value for 'default_bundle', so there needs to be two vocabularies in order to test that the 'default_bundle' value correctly appears in the UI. 